fdbserver/Knobs.cpp

```diff
@@ -246,7 +246,7 @@ ServerKnobs::ServerKnobs(bool randomize, ClientKnobs* clientKnobs) {
 	init( SPRING_CLEANING_MAX_VACUUM_PAGES,                      1e9 ); if( randomize && BUGGIFY ) SPRING_CLEANING_MAX_VACUUM_PAGES = deterministicRandom()->coinflip() ? 0 : deterministicRandom()->randomInt(1, 1e4);
 
 	// KeyValueStoreMemory
-	init( REPLACE_CONTENTS_BYTES,                                1e5 ); if( randomize && BUGGIFY ) REPLACE_CONTENTS_BYTES = 1e3;
+	init( REPLACE_CONTENTS_BYTES,                                1e5 );
```

Comment:
If "we cannot buggify replace content bytes because it takes too long to recovery when the txnStateStore is too large",
should we try to set this `REPLACE_CONTENTS_BYTES ` Knob to a random large, say a random value in [1e5, 1e6] or in [1e4, 1e6],  just to make sure setting it to a different large value won't break the correctness?

Reply:
I do not think larger value provide too much extra test coverage, i think we get all of the coverage we need with the new large txnStateStore simulation test